src/backend/embedding/extract_file/extract_file.py

```diff
@@ -329,7 +340,10 @@ def extract_pdf_content(self, customer_guid: str, file_path: str, filename: str)
 
                     # Append the result for this page (with metadata and content)
                     results.append({
```

Comment:
We are repeating results.append for every file

Can we make it a function and call from every file extraction so that we avoid duplicate code?